src/main/java/com/example/demo/dao/MatchDAO.java

```diff
@@ -0,0 +1,63 @@
+package com.example.demo.dao;
+
+import com.example.demo.configuration.DBConfig;
+import com.example.demo.model.Match;
+import lombok.AccessLevel;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.FieldDefaults;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@RequiredArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class MatchDAO {
+    static int id;
+
+    static {
+        id = 1;
+    }
+
+    public static void save(List<Match> matches) {
+        try {
+            PreparedStatement preparedStatement =
+                    DBConfig.getConnection().prepareStatement("insert into matches values(?,?,?,?,?)");
```

Comment:
SQL code should be written in this way:
```
INSERT INTO matches VALUES(?);
```

SQL commands are capitalized, your variables are in lower case.